tests/python_test.py

```diff
@@ -284,3 +284,11 @@ def test_write_resource_invalid_align_sector(
     with pytest.raises(ValueError):
         sanlock.write_resource(
             "ls_name", "res_name", disks, align=align, sector=sector)
+
+
+def test_write_resource_invalid_path(tmpdir, sanlock_daemon):
+    path = str(tmpdir.join("resources"))
+    disks = [path]
```

Comment:
We don't need a real path for this test, so we can do:

    disk = ["not a tuple"]